src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>The second picture fails to load on case-sensitive platforms because the actual file is <code>mdo.PNG</code> and not <code>mdo.png</code> .</p><p> </p><p>More generally, having upper case extensions mixed with lower case ones is inconsistent. I suggest putting them all in lower cases. I think a global textual search/replace in ypnb files will allow to easily take these changes into account.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Changed in 3a141cc1

Reply:
<p>The file is still named <code>mdo.PNG</code> (but all other files have a lowercase extension now).</p><p> </p>

Reply:
So ...
On GitHub (and in GitHub Desktop) it indeed appears as a .PNG but locally it is a .png 
![image](https://user-images.githubusercontent.com/66668699/191201042-08e242cc-5895-4fa6-bb92-a31b6b8c3b78.png)

Reply:
It worked !😆
Corrected in 0f83f585